### DISCUSSION TITLE
Install Issue 500 Internal Server Error, need help really!

### DISCUSSION THREAD

**Discussion by zentery** [chamilo/chamilo-lms#5874] [General]
When I installed the chamilo1.11.8 version, I unzipped the source code and installed it. The main page: Step 2 met the requirements, Step 3 had already created the database, and the last page was to fill in the information for installation. 
![3](https://github.com/user-attachments/assets/43075a19-e757-4848-8e14-3fab68cc8402)
![4](https://github.com/user-attachments/assets/85dc481a-d194-4ff5-803c-865540afa31c)
![5](https://github.com/user-attachments/assets/60a7b6cb-79fb-4a27-b0c6-7f92112c1966)

However, when I clicked Install, the page responded for a while and returned: 
![6](https://github.com/user-attachments/assets/52dc8db0-6f49-4ba5-a752-14cab16cacc2)

How should I solve this problem? I have tried three 1.11 release series. Can anyone help me? Thanks a lot!

**Comment by zentery** [comment]
@maher-nakesh During the installation of this software, there is no option to select a compiler. Sorry, I don't quite understand what you mean.

**Comment by Kaneda-1** [comment]
Hello.  Check the server error file error_log or similar.

**Comment by ywarnier** [comment]
@zentery You mention 1.11.8, the screenshots suggest it's 1.11.16, and the latest stable version is 1.11.26... so... which is it ?

The 500 Internal error points directly to something that would be in your error_log file. Given you're on Windows apparently, it 
would be good additional information to tell us if this is IIS or Apache, and if you are running WAMP or something like this.

If Apache, make sure the "Rewrite" module of Apache is enabled and check the installation guide to find the little block about rewrite
```
		AllowOverride all
                Require all granted
		Order allow,deny
		allow from all
```
These instructions may vary a bit depending on your version of Apache.

**Comment by zentery** [comment]
@ywarnier  Thanks for your answer. I am currently using version 1.11.16, using a windows system, phpstudy, apache2.4.39 and php7.4.3. I have already enabled the apache rewrite module and added the rewrite rule at the end, but it still returns 500.
![9](https://github.com/user-attachments/assets/907a0bba-5ff0-4920-9c71-93f7eb440454)

**Comment by ywarnier** [comment]
Did you have the opportunity to check your error_log?
The "500" error is a code to avoid showing the details of the error publicly, but the details are usually in the PHP or Apache error_log (it depends on your configuration)
